TheMovieDB/TheMovieDBTests/TestsNetworkProvider.swift

```diff
@@ -9,39 +9,39 @@ import XCTest
 @testable import TheMovieDB
 
 class TestsNetworkProvider: XCTestCase {
-    
-    var movies: [Movie] = []
-    
-    var imageView: UIImageView = UIImageView()
-    
-    let networkProvider = NetworkProvider()
-    
+                
     func testGetMovies() throws {
         
-        networkProvider.getMovies(completion: { [weak self] result in
+        let expect = expectation(description: "Download should have succeded.")
+                
+        Facade.shared.getMovies(completion: { result in
+            
+            XCTAssertNotNil(result)
+            XCTAssertEqual(result[0].title, "Dilwale Dulhania Le Jayenge")
+            XCTAssertEqual(result[3].originalLanguage, "en")
             
-            self?.movies = result
+            expect.fulfill()
         })
-        
-        XCTAssertNotNil(movies)
+        waitForExpectations(timeout: 5) { (error) in XCTAssertNil(error, "Test timed out")}
     }
     
     func TestGetImage() throws {
         
-        networkProvider.getImage(url: movies[0].imagePath ?? "", completion: { [weak self] resultOne in
+        let expect = expectation(description: "Download should have succeded.")
+        
+        var movies: [Movie] = []
+        
+        Facade.shared.getMovies(completion: { result in
             
-            DispatchQueue.main.async {
-                self?.imageView.image = resultOne
-            }
-            XCTAssertNotNil(resultOne)
+            movies = result
         })
```

Comment:
You don't have to download again all the data just take one path from the data there and use it in this test